zipline/examples/__init__.py

```diff
@@ -0,0 +1,17 @@
+from glob import glob
+from importlib import import_module
+import os
+
+for f in os.listdir(os.path.dirname(__file__)):
+    if not f.endswith('.py') or f == '__init__.py':
+        continue
+    modname = f[:-len('.py')]
+    globals()[modname] = import_module('.' + modname, package=__name__)
```

Comment:
ah, I wanted to be able to use `dir(zipline.examples)` to find the examples in the tests and as a user